proposals/flow-vs-typescript.md

```diff
@@ -0,0 +1,69 @@
+# Flow vs TypeScript
+
+Flow and TypeScript are both fantastic ways to limit technical debt in your JavaScript projects, allowing you to, among other things, evolve your codebase with confidence.
+They are similar tools that focus on static types checking.
+
+# Flow
+
+## Pros:
+
+1. As you're using React, we might prefer Flow because it easily integrates with Babel and other infrastructure.
+
+2. Flow is known to be a simpler language than TypeScript due to it’s out of the box utility. A developer would have to learn TypeScript syntax e.g. Type Annotation and how to modify the code.
+
+3. Eslint plugins for React and React Native provide valuable static analysis and seem to be superior to their tslint counterparts.
+
+4. As we are using PropTypes, It will be easier to switch to Flow.
+
+## Cons:
+
+1. New project, what might be cause a lot of changes, (errors) when we will be upgarding to the newest verion.
+
+2. Not all utility types from Flow are documented. This might be fixed soon though.
+
+# TypeScript
+
+## Pros:
+
+1. Larger community: Because it is an older project its community is larger.
+
+2. Faster then flow.
+
+3. Upfront compile errors: This can probably be configured in both TypeScript and Flow, but having upfront compilation errors instead of a non mandatory type check seems to save time when testing your code by preventing you from running code that is bogus anyways.
+
+4. More solid type checking and less changes when upgrade to newest version.
+
+## Cons:
+
+1. Lesser React support because its great for Angular project. But nowdays its getting better and better.
+
+2. Less flexible then Flow.
+
+#Key Differences Between TypeScript vs flow
```

Comment:
This heading is broken, needs a space after the hash

Reply:
ok